llvm/test/MC/RISCV/rv64p-valid.s

```diff
@@ -1,39 +1,102 @@
-# RUN: llvm-mc %s -triple=riscv64 -mattr=+experimental-p -M no-aliases -show-encoding \
+# RUN: llvm-mc %s -triple=riscv64 -mattr=+experimental-p -riscv-no-aliases -show-encoding \
```

Comment:
Why change this to `-riscv-no-aliases`? seems like all other line still using `-M no-aliases`?